src/app/view/components/dot-toolbar/components/dot-my-account/dot-my-account.component.spec.ts

```diff
@@ -207,13 +223,35 @@ describe('DotMyAccountComponent', () => {
         expect(comp.dialogActions.accept.disabled).toBe(true);
     });
 
+    it(`should call to add starter method in account service`, async () => {
+        spyOn<any>(accountService, 'addStarterPage').and.returnValue(of({ entity: {} }));
+        fixture.detectChanges();
+        comp.showStarter = true;
+        fixture.detectChanges();
+        const toggleShowStarter = de.query(By.css('#dot-my-account-show-starter'));
+        toggleShowStarter.triggerEventHandler('onChange', {});
+        expect(accountService.addStarterPage).toHaveBeenCalledTimes(1);
```

Comment:
You are not testing the change event you add to the html. That's is the tests not changing the `showStarter` manually.